packages/postcss-merge-longhand/src/lib/validateWsc.js

```diff
@@ -0,0 +1,51 @@
+import colorNames from 'css-color-names';
+
+const widths = ['thin', 'medium', 'thick'];
+const styles = ['none', 'hidden', 'dotted', 'dashed', 'solid', 'double', 'groove', 'ridge', 'inset', 'outset'];
+const colors = Object.keys(colorNames);
+
+export function isStyle (value) {
+    return value && !!~styles.indexOf(value);
+}
+
+export function isWidth (value) {
+    return value && !!~widths.indexOf(value) || /^(\d+(\.\d+)?|\.\d+)(\w+)?$/.test(value);
+}
+
+export function isColor (value) {
+    if (!value) {
+        return false;
+    }
+
+    if (/rgba?\(/.test(value)) {
+        return true;
+    }
+
+    if (/hsla?\(/.test(value)) {
+        return true;
+    }
+
+    if (/#([0-9a-zA-Z]{6}|[0-9a-zA-Z]{3})/.test(value)) {
+        return true;
+    }
+    if (value === 'transparent') {
+        return true;
+    }
+
+    if (value === 'currentColor') {

```

Comment:
All values from list can be in UPPERCASE format, css is case insensitivity (in many cases), don't forget this